sdk/keyvault/azure-security-keyvault/azure/security/keyvault/secrets/README.md

```diff
@@ -0,0 +1,252 @@
+# Azure Key Vault Secret client library for Python
+Azure Key Vault is a cloud service that provides a secure storage of secrets, such as passwords and database connection strings. Secret client library allows you to securely store and tightly control the access to tokens, passwords, API keys, and other secrets. This library offers operations to create, retrieve, update, delete, purge,backup, restore and and list the secrets and its versions.
+
+Use the secret client library to create and manage secrets.
+
+[Source code](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-security-keyvault/azure/security/keyvault/secrets) | [Package (Pypi)](TODO) | [API reference documentation](TODO) | [Product documentation](TODO)
+## Getting started
+### Install the package
+Install the Azure Key Vault client library for Python with [pip](https://pypi.org/project/pip/):
+
+`$ pip install azure-security-keyvault
+`
+
+#### Prerequisites
+* An [Azure subscription](https://azure.microsoft.com/free/).
+* Python 2.7, 3.4 or later to use this package.
+* An existing Key Vault. If you need to create a Key Vault, you can use the [Azure Cloud Shell](https://shell.azure.com/bash) to create one with this Azure CLI command:
```

Comment:
Add: Replace `<resource-group-name>` and `<key-vault-name>` with your own, unique names.

Reply:
Do we also want to replace the placeholders with unique names [here|https://github.com/Azure/azure-sdk-for-python/pull/5714/files/36fb66e6423926928b04ff0a6438326b7337acf3#diff-afd5dbe514ac682084c8b3a34c59af92R29] and [here|https://github.com/Azure/azure-sdk-for-python/pull/5714/files/36fb66e6423926928b04ff0a6438326b7337acf3#diff-afd5dbe514ac682084c8b3a34c59af92R32]

Reply:
I meant to just add a sentence saying all of that so that people know which part exactly they need to change (there is a bit of confusion around <> sometimes) :)